site/content/en/docs/Advanced/multi-cluster-allocation.md

```diff
@@ -96,7 +96,7 @@ To enable multi-cluster allocation, set `multiClusterSetting.enabled` to `true`
 NAMESPACE=<namespace>
 FLEET_NAME=<fleet name>
 
-curl https://${EXTERNAL_IP}:443/v1alpha1/gameserverallocation \
+curl https://${EXTERNAL_IP}:443/v1alpha/gameserverallocation \
```

Comment:
@pooneh-m Uh oh, I screwed up here. I meant to revert back to v1alpha1 but used v1alpha instead.

Reply:
Thanks for the catch. I'll fix it with my change.